scanner/src/main/kotlin/scanners/scanoss/ScanOssConfig.kt

```diff
@@ -40,15 +43,42 @@ internal data class ScanOssConfig(
         /** Name of the configuration property for the API key. */
         const val API_KEY_PROPERTY = "apiKey"
 
+        /** Name of the configuration property for the list of ignored file suffixes. */
+        const val IGNORED_FILE_SUFFIXES_PROPERTY = "ignoredFileSuffixes"
+
+        /** The default list of ignored file extension. */
+        private val IGNORED_FILE_EXTENSIONS = listOf(
+            ".1", ".2", ".3", ".4", ".5", ".6", ".7", ".8", ".9", ".ac", ".adoc", ".am", ".asciidoc", ".bmp", ".build",
+            ".c9", ".c9revisions", ".cache", ".cfg", ".chm", ".class", ".cmake", ".cnf", ".conf", ".config",
+            ".contributors", ".copying", ".cover", ".coverage", ".crt", ".csproj", ".css", ".csv", ".dat", ".data",
+            ".doc", ".docx", ".dotcover", ".dtd", ".dts", ".dtsi", ".dump", ".editorconfig", ".egg", ".eot", ".eps",
+            ".gdoc", ".gem", ".geojson", ".gif", ".glif", ".gml", ".gmo", ".gradle", ".guess", ".hex", ".htm", ".html",
+            ".ico", ".iml", ".in", ".inc", ".info", ".ini", ".ipynb", ".iws", ".jpeg", ".jpg", ".json", ".jsonld",
+            ".lcov", ".lock", ".log", ".lst", ".m4", ".manifest", ".map", ".markdown", ".md", ".md5", ".meta", ".mk",
+            ".mxml", ".o", ".otf", ".out", ".pbtxt", ".pdb", ".pdf", ".pem", ".phtml", ".pickle", ".pid", ".plist",
+            ".plt", ".png", ".po", ".pot", ".ppt", ".prefs", ".properties", ".pyc", ".qdoc", ".result", ".rgb", ".rst",
+            ".scss", ".sha", ".sha1", ".sha2", ".sha256", ".sln", ".spec", ".sql", ".sub", ".svg", ".svn-base", ".tab",
+            ".template", ".test", ".tex", ".tiff", ".toml", ".ttf", ".txt", ".utf-8", ".vim", ".wav", ".wfp", ".whl",
+            ".woff", ".xht", ".xhtml", ".xls", ".xlsx", ".xml", ".xpm", ".xsd", ".xul", ".yaml", ".yml"
+        )
+
+        /** The default list of ignored file suffixes. */
+        val IGNORED_FILE_SUFFIXES = IGNORED_FILE_EXTENSIONS + listOf(
+            "-doc", "authors", "changelog", "config", "copying", "ignore", "license", "licenses", "manifest", "news",
+            "notice", "readme", "sqlite", "sqlite3", "swiftdoc", "texidoc", "todo", "version"
```

Comment:
Should these be contained in the `reference.yml` as well? Or maybe go the other direction and reduce the list in `reference.yml` to just a few entries, as we don't use it to showcase default configuration but only configuration options.

Reply:
In the latest iteration I went for using the deprecated Java client's values as reference values, and the maintained Python client's values as default values. That way there is at least some sense in the reference values for those users who want to stick to the original values from the Java client, as they could easily copy them over.